src/components/modules/crossBlockSelection.ts

```diff
@@ -17,6 +17,19 @@ export default class CrossBlockSelection extends Module {
    */
   private lastSelectedBlock: Block;
 
+  /**
+   * Module preparation
+   *
+   * @returns {Promise}
+   */
+  public async prepare(): Promise<void> {
+    const { Listeners } = this.Editor;
+
+    Listeners.on(document, 'mousedown', (event: MouseEvent) => {
```

Comment:
are you sure that this event should belong to the document, instead of editor wrapper?

Reply:
yes, I've tested and it didn't work on editor's wrapper. Maybe @gohabereg could help

Reply:
@gohabereg answer here pls